sklearnex/tests/test_memory_usage.py

```diff
@@ -94,7 +94,10 @@ def remove_duplicated_estimators(estimators_list):
     return estimators_map.values()
 
 
-BANNED_ESTIMATORS = ("TSNE",)  # too slow for using in testing on common data size
+BANNED_ESTIMATORS = (
+    "IncrementalLinearRegression",  # TODO fix memory leak issue
```

Comment:
I am very happy to see the comment here. As we discussed, make sure to send me the ticket.